README.md

```diff
@@ -20,9 +20,11 @@ To get started, see the following resources:
 * [Amazon Chime](https://aws.amazon.com/chime)
 * [Amazon Chime SDK Security](https://aws.amazon.com/blogs/business-productivity/understanding-security-in-the-amazon-chime-application-and-sdk/)
 * [Amazon Chime SDK Pricing](https://aws.amazon.com/chime/pricing/#Chime_SDK_)
+* [Amazon Chime SDK for JavaScript Supported browser](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-browsers)
 * [Amazon Chime SDK Developer Guide](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html)
 * [Amazon Chime SDK API Reference](http://docs.aws.amazon.com/chime/latest/APIReference/Welcome.html)
 * [Amazon Chime SDK for JavaScript Documentation](https://aws.github.io/amazon-chime-sdk-js)
+* [Amazon Chime SDK Technical Blogs](https://aws.amazon.com/blogs/business-productivity/tag/amazon-chime-sdk/)
```

Comment:
Could we move this up between supported browser and developer guide? I think this list makes the most sense when ordered from highest-level to lowest-level.

Reply:
Updated